chandra_aca/star_probs.py

```diff
@@ -267,9 +257,84 @@ def acq_success_prob(date=None, t_ccd=-19.0, mag=10.0, color=0.6, spoiler=False,
     return probs[0] if is_scalar else probs
 
 
+def spline_model_acq_prob(mag=10.0, t_ccd=-12.0, color=0.6, halfwidth=120,
+                          probit=False, return_p_success=False):
+    """
+    """
+    try:
+        from scipy.interpolate import CubicSpline
+    except ImportError:
+        from .cubic_spline import CubicSpline
```

Comment:
Is there a local copy of cubic_spline somewhere (not in version control) for this code to work?

Reply:
Yes, good catch.